scripts/superdesk/editor/spellcheck/spellcheck.js

```diff
@@ -91,16 +191,20 @@ function SpellcheckService($q, api, dictionaries) {
                 tree = document.createTreeWalker(node, NodeFilter.SHOW_TEXT);
```

Comment:
Not sure if we care, but filter doesn't seem to be option in IE
https://developer.mozilla.org/en-US/docs/Web/API/Document/createTreeWalker